UI/src/application.js

```diff
@@ -1,4 +1,5 @@
 (function(application) {
+  var defaultLocation, startupAssetTypeId;
```

Comment:
Won't these variables declared to be global in this way? Is this right?

Reply:
Those variables were already used globally in the JS code, I just declared them initially to avoid grunt errors

Reply:
Understood.